tests/test_formatting.py

```diff
@@ -19,7 +20,7 @@
         ("{level.icon}", lambda r: r == "🐞"),
         ("{file}", lambda r: r == "test_formatting.py"),
         ("{file.name}", lambda r: r == "test_formatting.py"),
-        ("{file.path}", lambda r: r == __file__),
+        ("{file.path}", lambda r: os.path.samefile(r, __file__)),
```

Comment:
Why do you think this is preferable? I feel the opposite. Checking using `os.path.samefile()` is less restrictive than using `==`. The test must fail if `{file.path}` is `"/home/user/../user/foo.txt"` while `"/home/user/foo.txt"` is expected, but using `os.path.samefile()` won't detect that.

Reply:
That unit test failed on my system. The path's letter cases is inconsistent in Python on Windows. Maybe it should be `os.path.normcase`.

Reply:
Thanks for the explanation! Yes using `os.path.normcase()` is a good idea.

Reply:
Should this be resolved independently from [this](#935) and #943 PRs?